src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -5005,7 +5005,13 @@ void AuraEffect::HandlePreventResurrection(AuraApplication const* aurApp, uint8
 
 void AuraEffect::HandlePeriodicTriggerSpellAuraTick(Unit* target, Unit* caster) const
 {
-    uint32 triggerSpellId = GetSpellInfo()->Effects[GetEffIndex()].TriggerSpell;
+    uint32 triggerSpellId = GetSpellInfo()->Effects[m_effIndex].TriggerSpell;
```

Comment:
what's the m_effIndex change about ?

Reply:
Nothing is just direct access instead call a method that return it.

Reply:
Sounds like an improvement to me. 👍

Reply:
Changed to match the code in other parts of file